changelog/6900.improvement.md

```diff
@@ -0,0 +1 @@
+Debug logs from `matplotlib` libraries are now hidden by default and are configurable with the `LOG_LEVEL_LIBRARIES` environment variable. 
```

Comment:
(unrelated to this line, but wanted to add it somewhere)

`examples/moodbot/tests/test_stories.md` doesn't sound like something that should be added, it is empty and not using yml

Reply:
@erohmensing I saw they were added as part of https://github.com/RasaHQ/rasa/pull/6902 . Can you comment on any reason for adding it in?

Reply:
nope, should be left out, it was part of my debugging that i accidentally committed!